README.md

```diff
@@ -7,6 +7,9 @@ It defines a set of foundational type classes (e.g. `Functor`, `Monad`) and corr
 a large number of data structures.
 
 [![Build Status](https://secure.travis-ci.org/scalaz/scalaz.png?branch=series/7.3.x)](https://travis-ci.org/scalaz/scalaz)
+[![IRC](https://img.shields.io/badge/chat-on%20freenode-brightgreen.svg)](irc://irc.freenode.net/scalaz)
```

Comment:
this could also be some online IRC client link.

Reply:
"online IRC client"... :no_good_man: 

`hrino gets slapped around a bit with a large trout`

Reply:
hey, I don't want it. I'm anticipating someone saying "but people won't be able to use it without a client installed!"